### PR TITLE
Update ENTSO-E parser to use new proxy

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Description

Changes ENTSO-E parser to query via a new proxy service.

### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser "zone_key"`

### CODE REVIEW

**Review by VIKTORVAV99 — APPROVED** [review]
Looks like it's working perfectly!

There was a reason the checklist had two boxes though 😉

**Review by VIKTORVAV99 — APPROVED** [review]
Looks like it's working perfectly!

There was a reason the checklist had two boxes though 😉

**Comment by VIKTORVAV99** [comment]
Looks like we are mocking the tokens in a test too, we need to update that.

File: parsers/ENTSOE.py [hunk 1]
```diff
@@ -508,31 +497,25 @@
...
-        response: Response = session.get(url, params=params)
-        if response.ok:
-            return response.text
-        else:
-            last_response_if_all_fail = response
-    # If we get here, all tokens failed to fetch valid data
-    # and we will check the last response for a error message.
+    token = get_token("ENTSOE_TOKEN")
+    params["securityToken"] = token
+    response: Response = session.get(ENTSOE_URL, params=params)
```
[the remark this answers is not in the document]
Comment (madsnedergaard) [reply parsers/ENTSOE.py:502]: `get_token()` will raise an error if value is None, so no thank you 😗

**Comment by madsnedergaard** [comment]
> Looks like it's working perfectly!
> 
> There was a reason the checklist had two boxes though 😉

Haha, I've always just read the "pnpm ..." and assumed it was only about frontend 😅 
Also thought we had pre-commit running for contrib, but seems like we never add `ruff` to the checks? What's missing from replacing isort and black completely there?

**Comment by VIKTORVAV99** [comment]
> > Looks like it's working perfectly!
> > There was a reason the checklist had two boxes though 😉
> 
> Haha, I've always just read the "pnpm ..." and assumed it was only about frontend 😅 Also thought we had pre-commit running for contrib, but seems like we never add `ruff` to the checks? What's missing from replacing isort and black completely there?

Pretty sure I uninstalled isort and ruff, I don't think we ever had all of them in there since they where so slow...

**Comment by madsnedergaard** [comment]
> > > Looks like it's working perfectly!
> > > There was a reason the checklist had two boxes though 😉
> > 
> > 
> > Haha, I've always just read the "pnpm ..." and assumed it was only about frontend 😅 Also thought we had pre-commit running for contrib, but seems like we never add `ruff` to the checks? What's missing from replacing isort and black completely there?
> 
> Pretty sure I uninstalled isort and black, I don't think we ever had all of them in there since they where so slow...

On the monorepo we still run isort and black on pre-commit, so maybe it's time to replace that :)

**Comment by VIKTORVAV99** [comment]
> > > > Looks like it's working perfectly!
> > > > There was a reason the checklist had two boxes though 😉
> > > 
> > > 
> > > Haha, I've always just read the "pnpm ..." and assumed it was only about frontend 😅 Also thought we had pre-commit running for contrib, but seems like we never add `ruff` to the checks? What's missing from replacing isort and black completely there?
> > 
> > 
> > Pretty sure I uninstalled isort and black, I don't think we ever had all of them in there since they where so slow...
> 
> On the monorepo we still run isort and black on pre-commit, so maybe it's time to replace that :)

Yeah, I noticed we where still using black, led to some frustration when there was minor differences between ruff and black and I had autosave and format on save on 😅.

But I can take care of the pre-commit hook in here, then we can start replacing things on the backend too.

**Comment by madsnedergaard** [comment]
Will merge and release this to production tomorrow so I can watch in case issues occur! :)

**Pull request merged**